resources/definitions/rigid3d_base.def.json

```diff
@@ -18,96 +18,107 @@
-            "generic_nylon",
-            "generic_pc",
-            "generic_petg",
-            "generic_pla",
-            "generic_pp",
-            "generic_pva",
-            "generic_tough_pla",
-            "generic_tpu",
+            "generic_abs", "generic_bam", "generic_cffcpe", "generic_cffpa", "generic_cpe", "generic_cpe_plus",
+            "generic_gffcpe", "generic_gffpa", "generic_hips", "generic_nylon", "generic_pc", "generic_petg",
+            "generic_pla", "generic_pp", "generic_pva", "generic_tough_pla", "generic_tpu",
+            "goofoo_abs", "goofoo_asa", "goofoo_bronze_pla", "goofoo_emarble_pla", "goofoo_esilk_pla", "goofoo_hips", "goofoo_pa", "goofoo_pa_cf", "goofoo_pc",
+            "goofoo_peek", "goofoo_petg", "goofoo_pla", "goofoo_pva", "goofoo_tpe_83a", "goofoo_tpu_87a", "goofoo_tpu_95a", "goofoo_wood_pla",
             "imade3d_petg_175", "imade3d_pla_175",
             "innofill_innoflex60_175",
-			"leapfrog_abs_natural", "leapfrog_epla_natural", "leapfrog_pva_natural",
+            "layer_one_white_pla", "layer_one_black_pla", "layer_one_dark_gray_pla",
+            "leapfrog_abs_natural", "leapfrog_epla_natural", "leapfrog_pva_natural",
             "octofiber_pla",
             "polyflex_pla", "polymax_pla", "polyplus_pla", "polywood_pla",
             "redd_abs", "redd_asa", "redd_hips", "redd_nylon", "redd_petg", "redd_pla", "redd_tpe",
             "structur3d_dap100silicone",
             "tizyx_abs", "tizyx_flex", "tizyx_petg", "tizyx_pla", "tizyx_pla_bois", "tizyx_pva",
-            "ultimaker_abs_black", "ultimaker_abs_blue", "ultimaker_abs_green", "ultimaker_abs_grey", "ultimaker_abs_orange", "ultimaker_abs_pearl-gold", "ultimaker_abs_red", "ultimaker_abs_silver-metallic", "ultimaker_abs_white", "ultimaker_abs_yellow",
-            "ultimaker_bam",
-            "ultimaker_cpe_black", "ultimaker_cpe_blue", "ultimaker_cpe_dark-grey", "ultimaker_cpe_green", "ultimaker_cpe_light-grey", "ultimaker_cpe_plus_black", "ultimaker_cpe_plus_transparent", "ultimaker_cpe_plus_white", "ultimaker_cpe_red", "ultimaker_cpe_transparent", "ultimaker_cpe_white", "ultimaker_cpe_yellow",
-            "ultimaker_nylon_black", "ultimaker_nylon_transparent",
-            "ultimaker_pc_black", "ultimaker_pc_transparent", "ultimaker_pc_white",
-            "ultimaker_pla_black", "ultimaker_pla_blue", "ultimaker_pla_green", "ultimaker_pla_magenta", "ultimaker_pla_orange", "ultimaker_pla_pearl-white", "ultimaker_pla_red", "ultimaker_pla_silver-metallic", "ultimaker_pla_transparent", "ultimaker_pla_white", "ultimaker_pla_yellow",
-            "ultimaker_pp_transparent",
-            "ultimaker_pva",
-            "ultimaker_tough_pla_black", "ultimaker_tough_pla_green", "ultimaker_tough_pla_red", "ultimaker_tough_pla_white",
-            "ultimaker_tpu_black", "ultimaker_tpu_blue", "ultimaker_tpu_red", "ultimaker_tpu_white",
+            "ultimaker_abs_black", "ultimaker_abs_blue", "ultimaker_abs_green", "ultimaker_abs_grey",
```

Comment:
The ultimaker profiles don't even need to be mentioned here, as this is a 1.75mm printer. They are filtered out automatically (DRY principle; Don't repeat yourself)

Reply:
Didn't knew that. Removed the material exclusions. Thank you.